py_mmd_tools/script/nc2mmd.py

```diff
@@ -71,7 +75,8 @@ def main(args=None):
         md = nc_to_mmd.Nc_to_mmd(str(file), output_file=outfile)
         md.to_mmd(
             add_wms_data_access=args.add_wms_data_access,
-            checksum_calculation=args.checksum_calculation
+            checksum_calculation=args.checksum_calculation,
+            collection=args.collection
```

Comment:
Need to test type, if only string (One collection) is to be allowed.

Also some changes need to be made in nc_to_mmd.to_mmd-function

Reply:
then it is ok.